scripts/db-entrypoint.sh

```
@@ -10,11 +10,25 @@ sed_vars () {
+        echo "Copying file $file after seding vars"
+        sed_vars "$file"
+    fi
 done
 
 cd ../entrypoint/ || exit
 
 for file in *.sql; do
-    mariadb -u root -p"$MYSQL_ROOT_PASSWORD" < "$file"
+    if [[ "$file" == *dump* ]]; then
+        filesize=$(stat -Lc%s "$file" 2>/dev/null || stat -Lf%z "$file" 2>/dev/null)
+        echo "Importing $file ($(numfmt --to=iec $filesize)) ..."
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)